python/eos/parameter.py

```diff
@@ -99,24 +99,40 @@ def _repr_html_(self):
 
 
     @staticmethod
-    def FromWCxf(wc):
+    def FromWCxf(w):
+        """
+        Imports Wilson coefficients into an eos.Parameters object from a wilson.Wilson object.
+
+        :param w: WET Wilson coefficients in the EOS basis.
+        :type w: wilson.Wilson
+        """
+
+        # extract wc data
+        wc = w.wc
+
+        if wc.basis != 'EOS':
+            raise ValueError('Wilson coefficients must be passed to this function in the EOS basis')
+
+        # Needs to be revisited, once EOS support WET-4 or WET-3 bases
+        if wc.scale != 4.2:
+            raise ValueError('Wilson coefficients must be passed to this function at the reference scale 4.2 GeV')
 
```

Comment:
Should we open an issue to do this automatically?

Reply:
Hmm, I kind of like that the user must be aware of this issue. It should not be hardship for the user to use match_run correctly.